### ISSUE TITLE
Clean up entrypoints/package config/Girder-Worker config

### ISSUE THREAD

**Issue by mgrauer** [girder/girder_worker#163]
This isn't related to the celery refactor necessarily, but would be good to be done after all of that churn settles down.

Additionally, making it so that pip installing girder-worker also sets the girder-worker config settings with the plugins.  This second step has bitten me and at least two others; it would be nice if the install process could be streamlined.

**Comment by kotfic** [comment]
This isn't really related to the refactor,  I am going to remove it from the milestone

**Comment by kotfic** [comment]
@mgrauer seems like there are a few different thing going on here?  Can we break this out into individual issues?

**Comment by mgrauer** [comment]
Yes certainly. I can't recall what the issue title is about, perhaps I was transcribing it from someone who remembers the details better?

I am interested in having the girder worker config settings include the plugins when they are passed to pip, if that is possible, so that installation won't require this additional step. I.e., when I pip install GW with certain plugins, those plugins should be included in the GW config file.

**Comment by kotfic** [comment]
Ah,  in that case I will mark this as Legacy,  and if you or anyone else remembers what the other components of the issues are they can raise new ones.
